lib/src/Dscp/Educator/Launcher/Runner.hs

```diff
@@ -15,7 +15,7 @@ import Dscp.Witness.Launcher.Runner (formWitnessContext)
 formEducatorContext :: EducatorParams -> EducatorResources -> IO EducatorContext
 formEducatorContext _ecParams EducatorResources{..} = do
     let _ecDB = _erDB
-    let _ecSecret = _erSecret
+    let _ecKeys = _erKeys
```

Comment:
It is too near to `let x = x`.

I'm still against `RecordWildCards`.

Not only it moves assignments away from records that receives them, but it also makes it possible to create `forall a. a` object (which is impossible with normal record assignment).

And, also. The code is supposed to be read by people, so lets not fill it with "connect property to corresponding record" riddles.

Reply:
Okay, here it really would look better without such `let`s